frontend/src/components/ShootDns/GDnsProvider.vue

```diff
@@ -28,17 +28,19 @@ SPDX-License-Identifier: Apache-2.0
         />
       </v-chip>
     </template>
-    <v-list class="pa-0">
+    <v-list min-width="300px">
```

Comment:
We usually do not use units for component widths.
```suggestion
    <v-list min-width="300">
```